app/src/main/java/org/simple/clinic/summary/PatientSummaryScreen.kt

```diff
@@ -799,43 +799,35 @@ class PatientSummaryScreen :
   }
 
   override fun showStatinAlert(statin: StatinModel) {
-    val statinAlertDescriptionText = buildString {
+    statinAlertDescription.text = buildString {
       append("${getString(R.string.statin_alert_patient)} ")
+
       if (statin.hasDiabetes) {
         append(String.format(getString(R.string.statin_alert_has_diabetes), statin.age.toString()))
+
+        if (statin.hasHadHeartAttack.xor(statin.hasHadStroke)) {
```

Comment:
Doesn't `xor` also return true when both are false? Can you confirm that case once?

Reply:
@msasikanth 

If both inputs are false , the output is false.
When one input is true and the other is false, the output is true.
When both inputs are true, the output is false.